.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,4 @@
+# Contributing
+
+Please take some time to get familiar with the [contributing guidelines](https://docs.cscs.ch/contributing/) before making your first contribution.
+Following the guidelines helps us keep the documentation consistent and as useful as possible for users.
```

Comment:
Maybe this sentence is not needed?
It is said in more than one spot in the contributing guide that is linked above.

Reply:
Yep, fair! Removed in https://github.com/eth-cscs/cscs-docs/pull/250/commits/400ea15074a837dcb028a3f421004a4be30748b7.

Would you like to see the same sentence removed from the welcome message as well? I quite like it there as it's posted only once and gives a (very brief) motivation for contributors to really read the guidelines. What do you think?